### ISSUE TITLE
Package Microsoft.AspNetCore.Blazor.Cli 0.9.0-preview4-19201-04 is not compatible with netcoreapp2.2

### ISSUE THREAD

**Issue by danroth27** [aspnet/AspNetCore#8999]
Repro steps:
- .NET Core 3.0 SDK: 3.0.100-preview4-011022
- Blazor VSIX: 16.0.19201.4
- Create new Blazor app
- Workaround: update csproj 

  ```xml
  <Project Sdk="Microsoft.NET.Sdk.Web">
  
    <PropertyGroup>
      <TargetFramework>netstandard2.0</TargetFramework>
      <RunCommand>dotnet</RunCommand>
      <RunArguments>blazor serve</RunArguments>
      <RestoreAdditionalProjectSources>
        https://dotnet.myget.org/F/aspnetcore-dev/api/v3/index.json;
        https://dotnet.myget.org/F/blazor-dev/api/v3/index.json;
      </RestoreAdditionalProjectSources>
      <LangVersion>7.3</LangVersion>
      <RazorLangVersion>3.0</RazorLangVersion>
      <AddRazorSupportForMvc>false</AddRazorSupportForMvc>
      <GenerateRazorAssemblyInfo>false</GenerateRazorAssemblyInfo>
    </PropertyGroup>
  
    <ItemGroup>
      <PackageReference Include="Microsoft.AspNetCore.Blazor" Version="0.9.0-preview4-19201-04" />
      <PackageReference Include="Microsoft.AspNetCore.Blazor.Build" Version="0.9.0-preview4-19201-04" PrivateAssets="all" />
  
      <DotNetCliToolReference Include="Microsoft.AspNetCore.Blazor.Cli" Version="0.9.0-preview4-19201-04" />
    </ItemGroup>
  
  </Project>
  ```
- Build and run

Actual result:

Package Microsoft.AspNetCore.Blazor.Cli 0.9.0-preview4-19201-04 is not compatible with netcoreapp2.2 (.NETCoreApp,Version=v2.2). Package Microsoft.AspNetCore.Blazor.Cli 0.9.0-preview4-19201-04 supports: netcoreapp3.0 (.NETCoreApp,Version=v3.0)

**Comment by danroth27** [comment]
@SteveSandersonMS

**Comment by placetobejohan** [comment]
I was able to solve this by checking the option **Use previews of the .Net Core SDK** under Tools --> Options --> Projects and Solutions --> .NET Core in VS 2019.

**Comment by SteveSandersonMS** [comment]
Fixed with #9028.
